src/lib/ngsi/ContextAttribute.h

```diff
@@ -63,6 +63,7 @@ typedef struct ContextAttribute
   bool                       found;                   // Not part of NGSI, used internally for CPr forwarding functionality (update case)
                                                       // It means attribute found either locally or remotely in providing application
 
+  bool                       skip;                    // In case of 'op=append' and the attribute already exists
```

Comment:
Maybe is better:

"Internal usage in mongoBackend: in case of 'op=append' and the attribute already exists"